src/scss/components/checkbox/mixins/_hxCheckboxControl.scss

```diff
@@ -61,24 +69,26 @@
 @mixin __checkboxControl--disabled($pseudo-state: null) {
   @if $pseudo-state == focus {
     &::-ms-check {
+      border-color: $gray-500;
       border-color: var(--hxCheckbox-disabled-bordercolor, $gray-500);
-      box-shadow: $focus-glow-disabled;
+      // box-shadow: $focus-glow-disabled; // TBD v2.0 specs
```

Comment:
another box shadow area to confirm

Reply:
Yes, we need clarification from design on box shadow for checkbox.